R/globals.R

```diff
@@ -84,14 +84,25 @@ globalVariables(c(
   "order2",
   "order3",
   "_unit",
+  "temp_DT",
+  "temp_from_var",
+  "temp_to_var",
+  "temp_match_flag",
+  "temp_dose_freq",
+  "temp_new_dose_no",
+  "temp_num_of_doses",
+  "temp_dose_multiplier",
+  "temp_day_difference",
   "tmp_analysis_date",
   "tmp_dose_date",
   "tmp_seq_var",
   "EXDOSFRQ",
   "tmp_ldose_dt",
```

Comment:
Is this something to do for later - update to temp_* in other .R files?  Make a separate issue?

Reply:
Yes, let's create a new issue to update this. I'd actually prefer `tmp_` over `temp_` but I guess either is fine as long as we are consistent.